features/apply_with_s3.feature

```diff
@@ -0,0 +1,106 @@
+Feature: Apply command
+
+  Background:
+    Given a file named "stack_master.yml" with:
+      """
+      stack_defaults:
+        s3:
+          bucket: my-bucket
+          region: us-east-1
+          prefix: cfn_templates/my-app
+      stacks:
+        us_east_1:
+          myapp_vpc:
+            template: myapp_vpc.rb
+            files:
+              - user_data.sh
+      """
+    And a directory named "parameters"
+    And a file named "parameters/myapp_vpc.yml" with:
+      """
+      KeyName: my-key
+      """
+    And a file named "parameters/myapp_web.yml" with:
+      """
+      VpcId: vpc-blah
+      """
+    And a directory named "templates"
+    And a file named "templates/user_data.sh" with:
+      """
+      #!/bin/bash
+      echo "HI"
+      """
```

Comment:
Should we looks for `files` in a different dir than `templates`? Maybe `templates/files/` or `files/` at the top level?